pkgs/applications/misc/parsec/generic.nix

```diff
@@ -23,34 +23,58 @@
 , vulkan-loader
 , gnome
 , zenity ? gnome.zenity
+, darwin
+, p7zip
+, libarchive
 }:
 
-stdenvNoCC.mkDerivation {
-  pname = "parsec-bin";
-  version = "150_93b";
+{ pname
+, version
+, url
+, hash
+}:
+
+let
+  inherit (stdenv) targetPlatform;
+  ARCH = if targetPlatform.isArch64 then "arm64" else "x64";
```

Comment:
this looks unused

Reply:
It is now that I look at it, it was from a template of a derivation. I will remove it.